modules/tinymce/src/core/main/ts/dom/DomSerializerImpl.ts

```diff
@@ -90,18 +91,22 @@ const DomSerializerImpl = (settings: DomSerializerSettings, editor?: Editor): Do
 
   const dom = editor && editor.dom ? editor.dom : DOMUtils.DOM;
   const schema = editor && editor.schema ? editor.schema : Schema(settings);
-  settings.entity_encoding = settings.entity_encoding || 'named';
-  settings.remove_trailing_brs = 'remove_trailing_brs' in settings ? settings.remove_trailing_brs : true;
 
-  const htmlParser = DomParser(settings, schema);
-  DomSerializerFilters.register(htmlParser, settings, dom);
+  const defaultedSettings: DomSerializerSettings = {
```

Comment:
Move this to the top of the function, so that we can have `Schema(defaultedSettings)` as well, the aim is to only use `defaultedSettings`.